### PR TITLE
Fix CI tests

### PR DESCRIPTION
**Pull request by adriendelsalle**
Description
--

- both `linux` and `osx` umamba C++ tests are silently failing because of a missing `gmock` dependency that should not be required
- `windows` mamba python tests are silently failing. No filter is set to avoid running micromamba python tests

### CODE REVIEW

File: test/downloadGTest.cmake.in [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# The full license is in the file LICENSE, distributed with this software. #
+############################################################################
+
+cmake_minimum_required(VERSION 2.8.2)
+
+project(googletest-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(googletest
+    GIT_REPOSITORY    https://github.com/JohanMabille/googletest.git
```
Comment (wolfv) [inline test/downloadGTest.cmake.in:17]: let's not include this. I really think we should just fix gtest package or add gmock.

Comment (adriendelsalle) [reply test/downloadGTest.cmake.in:17]: I also prefer using the packaged version of gtest. We should fix upstream, I don't like having a dep to something we don't need

Comment (adriendelsalle) [reply test/downloadGTest.cmake.in]: It's now fixed without installing from sources. Just modified to use `add_library` instead of `find_package`.

**Comment by wolfv** [comment]
gr8!

**Pull request merged**